tests/integration/tasks/test_text_summary_task.py

```diff
@@ -7,7 +7,7 @@ class TestTextSummaryTask:
     @pytest.fixture(
         autouse=True,
         params=StructureTester.TEXT_SUMMARY_TASK_CAPABLE_PROMPT_DRIVERS,
-        ids=StructureTester.prompt_driver_id_fn,
+        ids=StructureTester.generate_prompt_driver_id,
```

Comment:
Why isn't this `prompt_driver_id_provider` or `prompt_driver_id_factory`?

Reply:
Because that's actually a class method, not a field. Methods should still be verb-noun.